src/components/common/Cards/main.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+
+export default class MainCard extends Component {
+  render() {
+    return (
+      <div className="left">
+        <div className="article-card">
+          <div className="avatar-wrapper">
```

Comment:
I would suggest to make this avatar wrapper a separate component since it's been used in many places

Reply:
@espoirMur, we have many components that will be reused, the aim of this PR was to create the bare minimum components to start with.  So, that should be implemented in the future.